src/cloudai/_core/test_scenario_parser.py

```diff
@@ -192,5 +209,7 @@ def _create_test_run(self, test_info: _TestRunTOML, normalized_weight: float) ->
             sol=test_info.sol,
             weight=test_info.weight * normalized_weight,
             ideal_perf=test_info.ideal_perf,
+            prologue=prologue if prologue is not None else TestScenario(name="default_prologue", test_runs=[]),
```

Comment:
TestRun's prologue/epilogue can be None, so you can just assign `prologue=prologue` here, we don't need defaults for these values.